evalml/demos/fraud.py

```diff
@@ -23,8 +25,9 @@ def load_fraud(n_rows=None, verbose=True, return_pandas=False):
                      target="fraud",
```

Comment:
We should update docstrings for these demo dataset methods! :D

Reply:
Great suggestion!